CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## Unreleased
 
-- [`sentry_dio`](https://pub.dev/packages/sentry_dio) is promoted to GA and not experimental anymore ([#914](https://github.com/getsentry/sentry-dart/pull/914))
+* Fix: addBreadcrumb throws on Android API < 24 because of NewApi usage ([#900](https://github.com/getsentry/sentry-dart/pull/900))
```

Comment:
```suggestion
* Fix: addBreadcrumb throws on Android API < 24 because of NewApi usage ([#923](https://github.com/getsentry/sentry-dart/pull/923))
```